app/models/service_ansible_playbook.rb

```diff
@@ -47,6 +47,16 @@ def postprocess(action)
 
   private
 
+  def manageiq_extra_vars
+    {
+      'MIQ_URL'          => MiqRegion.my_region.remote_ws_url,
+      'MIQ_TOKEN'        => Api::UserTokenService.new.generate_token(evm_owner.userid, 'api'),
+      'MIQ_SERVICE'      => id,
```

Comment:
@bzwei 
This should use the slug on the service object
Should we all also add Service Action which will come in handy when the same playbook is used for Provisioning, Reconfig and Retirement